src/test/java/org/elasticsearch/bwcompat/OldIndexBackwardsCompatibilityTests.java

```diff
@@ -70,6 +84,76 @@ public static void initIndexes() throws Exception {
         Collections.sort(indexes);
     }
     
+    @AfterClass
+    public static void tearDownStatics() {
+        indexes = null;
+        indicesDir = null;
+    }
+    
+    @Override
+    public Settings nodeSettings(int ord) {
+        return ImmutableSettings.builder()
+            .put(Node.HTTP_ENABLED, true) // for _upgrade
+            .put(MergePolicyModule.MERGE_POLICY_TYPE_KEY, NoMergePolicyProvider.class) // disable merging so no segments will be upgraded
+            .build();
+    }
+    
+    void setupDataLoadingNodes() throws Exception {
+        Path dataDir = newTempDirPath(LifecycleScope.SUITE);
+        indicesDir = dataDir.resolve(internalCluster().getClusterName()).resolve("nodes/0/indices/");
```

Comment:
Sorry, I got confused because I have a branch I've been working on which fixes this writing to CWD. We shouldn't have any code relying on CWD, and with my change, each node will have its own isolated base path.

> Not starting an extra node is a bonus.

There is no extra node? The data nodes started as part of the cluster are just for replicas. 

Is this disagreement on test style really worth blocking this improvement? Perhaps in the future I will agree, but we should get the tests faster now, and work on perfection later.